wallet/tickets.go

```diff
@@ -25,6 +26,27 @@ func sliceContainsHash(s []chainhash.Hash, h chainhash.Hash) bool {
 	return false
 }
 
+// GenerateVoteTx returns a hash
+func (w *Wallet) GenerateVoteTx(blockHash *chainhash.Hash, height int64, sstxHash *chainhash.Hash, voteBits stake.VoteBits) (*wire.MsgTx, error) {
+	var msgtx *wire.MsgTx
+	err := walletdb.Update(w.db, func(tx walletdb.ReadWriteTx) error {
+		stakemgrNs := tx.ReadWriteBucket(wstakemgrNamespaceKey)
+		addrmgrNs := tx.ReadBucket(waddrmgrNamespaceKey)
+
+		shit, err := w.StakeMgr.GenerateVoteTx(stakemgrNs, addrmgrNs, blockHash, height, sstxHash, voteBits)
+		if err != nil {
+			return err
+		}
+		msgtx = shit
```

Comment:
Mutating outer variable doesn't look right to me. But I guess this can only be fixed with changing `walletdb.Update` API.